parser.y

```diff
@@ -5519,6 +5522,45 @@ NotKeywordToken:
 |	"LEARNER"
 |	"VOTER"
 
+/************************************************************************************
+ *
+ *  Call Statements
+ *
+ **********************************************************************************/
+CallStmt:
+	"CALL" ProcedureCall
+	{
+		$$ = &ast.CallStmt{
+			Procedure: $2.(*ast.FuncCallExpr),
+		}
+	}
+
+ProcedureCall:
+	identifier
+	{
+		$$ = &ast.FuncCallExpr{
+			FnName: model.NewCIStr($1),
+			Args:   []ast.ExprNode{},
+		}
+	}
+|	Identifier '.' Identifier
+	{
+		var tp ast.FuncCallExprType
+		if isInTokenMap($3) {
+			tp = ast.FuncCallExprTypeKeyword
+		} else {
+			tp = ast.FuncCallExprTypeGeneric
+		}
+
```

Comment:
this should always be `FuncCallExprTypeGeneric`.

the `FuncCallExprTypeKeyword` was introduced to distinguish between built-in functions like `substr('x', 'y')` from user-defined functions like `` `substr`('x', 'y')`` (see #876). since there is no built-in stored procedures in a universal database, there's no need to use `FuncCallExprTypeKeyword` either.

Reply:
fixed